### DISCUSSION TITLE
Dynamic Links / Deferred Deep Linking

### DISCUSSION THREAD

**Discussion by VedKathe** [supabase/supabase#29523] [Feature Requests]
Does Supabase plan to support Dynamic Links like Firebase or Branch.io?

This is important because the same link should go to a webpage if the app is not installed or to a deep link within the app if it is installed.

It's just these "little" things that I miss about Firebase. 

![image](https://github.com/user-attachments/assets/2092fd4c-5416-40d2-96d2-2e20ee0b224d)

**Comment by anisikram** [comment]
Since firebase will deprecate dynamic links it will be nice to have a such feature in supabase

**Comment by SenchoPoro42** [comment]
I concur, this feature would be so cool to have for premium and seamless applications.
